code/game/machinery/doors/airlock.dm

```diff
@@ -1066,4 +1067,86 @@ About the new airlock wires panel:
 		src.unlock()
 		src.open()
 		src.lock()
-	return
\ No newline at end of file
+	return
+
+/obj/machinery/door/airlock/server_interface(var/list/argslist)
+	if(argslist && argslist.len >= 2)
+		var/activate = argslist[2]
+		switch(argslist[1])
+			if("idscan")
+				if(isWireCut(AIRLOCK_WIRE_IDSCAN))

```

Comment:
don't use empty if statements for an else-if chain
do 
```
if(!isWireCut(AIRLOCK_WIRE_IDSCAN))
    if(activate && aiDisabledIdScanner)
			aiDisabledIdScanner = 0
	else if(!activate && !aiDisabledIdScanner)
			aiDisabledIdScanner = 1
```
that goes for all of these